js/custom-script.js

```diff
@@ -311,7 +311,7 @@ $(function(){
 
       // RSD or christmas
       // Like for Books, if Artist is Beach Boys we load all items. Separate from books bc they use byArtist instead of aboutArtist
-      else if( selectedCategory == 'rsd' || 'christmas'){
+      else if( selectedCategory == 'rsd' || selectedCategory == 'christmas'){
```

Comment:
```suggestion
      else if(selectedCategory == 'rsd' || selectedCategory == 'christmas'){
```

minor: extra space after paren